src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,45 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+  die() {
+    const index = Animal.alive.indexOf(this);
+
+    if (index > -1) {
+      Animal.alive.splice(index, 1);
```

Comment:
It is preferred to filter animals with health less than or equal to 0 from `Animal.alive` array using Array's `filter` method instead of `splice`. Consider refactoring this part to use `filter` for consistency with the task checklist.